server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/pipeline_configs/pipeline_config_widget.js.msx

```diff
@@ -140,10 +147,7 @@ define([
 
         var pipelineFlowWidget = function () {
           if(!s.isBlank(ctrl.pipeline().template())) {
```

Comment:
IIRC, there's no blank check in lodash. If you're checking on strings, underscore.string is probably a better choice than lodash.